src/slic3r/GUI/Tab.cpp

```diff
@@ -266,7 +266,7 @@ void Tab::create_preset_tab()
     set_tooltips_text();
 
     add_scaled_button(m_top_panel, &m_undo_btn,        m_bmp_white_bullet.name());
-    add_scaled_button(m_top_panel, &m_undo_to_sys_btn, m_bmp_white_bullet.name());
+    //add_scaled_button(m_top_panel, &m_undo_to_sys_btn, m_bmp_white_bullet.name());
```

Comment:
@Noisyfox what is this change for? I don't quit follow the changes in this file.

Reply:
https://github.com/SoftFever/OrcaSlicer/issues/6261#issuecomment-2284606337
For fixing this. This button is never used, and somehow on Linux it's shown, but not on Windows. So I think it's better to remove it completely.